changelog.json

```diff
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "1.5.0-beta1": [
+      "[New] Repository switcher has a convenient \"Add\" button to add other repositories - #5878",
+      "[New] macOS tray icon now supports drag-and-drop to add local repositories - #5048",
+      "[Added] Keyboard shortcut for \"Rename Branch\" menu item - #5964. Thanks @agisilaos!",
+      "[Added] Commit file list now has \"Copy File Path\" context menu action - #2944. Thanks @Amabel!",
+      "[Fixed] Launching Desktop from browser always asks to clone repository - #5913",
+      "[Fixed] Publish dialog displayed on push when repository is already published - #5936",
+      "[Improved] Sanitize repository name before publishing to GitHub - #3090. Thanks @Daniel-McCarthy!",
+      "[Improved] Avoid optional locks when checking status of repository - #5952",
```

Comment:
i'm not sure this should be a release note. or we maybe we just need to rephrase it to highlight the user-facing part.

Reply:
```suggestion
      "[Improved] Getting a repository's status only blocks other operations when absolutely necessary - #5952",
```